src/Tests/AddThisFieldWebTestBase.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/**
+ * @file
+ * Definition of Drupal\addthis\Tests\AddThisFieldWebTestBase.
+ *
+ * This code was taken from the field_example module in the
+ * D8 examples module. Would be awesome if something like this was provided
+ * by core.
+ */
+
+namespace Drupal\addthis\Tests;
+
+use Drupal\Core\Session\AccountInterface;
+use Drupal\simpletest\WebTestBase;
+
+class AddThisFieldWebTestBase extends WebTestBase {
```

Comment:
What is going on with this? It seems really bizzare to me - like your testing fields and content types. Not clear on why or how this is needed